freqtrade/arguments.py

```diff
@@ -335,12 +335,25 @@ def hyperopt_options(parser: argparse.ArgumentParser) -> None:
             metavar='INT',
         )
 
+    @staticmethod
+    def list_exchanges_options(parser: argparse.ArgumentParser) -> None:
+        """
+        Parses given arguments for the list-exchanges command.
+        """
+        parser.add_argument(
+            '-1',
```

Comment:
maybe add a `--one-column` here? 
also, maybe it's better to call it `-l` - as "list"? 

having a number as argument was surprising as i've not seen it around so far (although it works just fine, technically)

Reply:
> having a number as argument was surprising as i've not seen it around so far (although it works just fine, technically)

Ermmm, `man ls`... ;)
This often (more or less) used in unix core utils to specify output in one column...

But, yes, it may be surprising and, plus, it's not recognized well from '-l'. Okay, I will change it to `--one-column`, it's a good name for this option. (Maybe make it `-1/--one-column`?)

Reply:
> (Maybe make it -1/--one-column?) 

sure

Reply:
> sure

Done. ;)